types/wordpress__components/button/index.d.ts

```diff
@@ -129,8 +129,8 @@ declare namespace Button {
          */
         variant?: ButtonVariant | undefined;
     }
-    interface AnchorProps extends BaseProps, HTMLProps<HTMLAnchorElement> {}
-    interface ButtonProps extends BaseProps, HTMLProps<HTMLButtonElement> {
+    interface AnchorProps extends BaseProps, Omit<HTMLProps<HTMLAnchorElement>, keyof BaseProps> {}
+    interface ButtonProps extends BaseProps, Omit<HTMLProps<HTMLButtonElement>, keyof BaseProps> {
```

Comment:
I think these should rather fix the incompatible types to create large mapped types. For example, `className` probably wants to be `HTMLProps<HTMLButtonElement>['className']`?

Reply:
Maybe? Maybe not? I have never used this library and never will, so I don't feel comfortable deciding on this and decided to make the smallest change.